### ISSUE TITLE
Restart ListItem Numbering

### ISSUE THREAD

**Issue by RLovelett** [PHPOffice/PHPWord#10]
Is there a way to restart the ListItem numbering (lettering)?

Using this code snippet:

``` php
$numberStyleList = array('listType' => \PHPWord_Style_ListItem::TYPE_NUMBER);
$alphStyleList = array('listType' => \PHPWord_Style_ListItem::TYPE_ALPHANUM);

$section->addListItem('One', 0, null, $numberStyleList);
$section->addListItem('Two', 0, null, $numberStyleList);
$section->addListItem('Alpha', 1, null, $alphStyleList);
$section->addListItem('Beta', 1, null, $alphStyleList);
$section->addListItem('Three', 0, null, $numberStyleList);
$section->addListItem('Charlie', 1, null, $alphStyleList);
```

Currently it would output this:

```
1. One
2. Two
   A. Alpha
   B. Beta
3. Three
   C. Charlie
```

I want to output this:

```
1. One
2. Two
   A. Alpha
   B. Beta
3. Three
   A. Charlie
```

Is there some other code call that needs to be made? Or is this a limitation of PHPWord?

**Comment by Kell** [comment]
Hey folks,
 i see the issue is closed but i don't see how list numbering is restarted just custom numbering styling was added?
I also found this https://stackoverflow.com/questions/34390893/phpword-and-listitemrun-reset-numbering , which seems to me that its still not possible in proper way and i need to make new style if i want a reset ?
Setting numbers manually isn't possible, right? for example to set the "counter" back to 1?
